en/docs/reference/product-apis/service-catalog-apis/service-catalog-v1/service-catalog-v1.yaml

```diff
@@ -0,0 +1,1089 @@
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+            "https://127.0.0.1:9443/api/am/service-catalog/v0/services/3784c2da-2f2a-4af9-8a43-ef207fb4d739/definition"'
+
+  #-------------------------------------------------------------------
+  # Retrieve the information of APIs that use a given service
+  #-------------------------------------------------------------------
+  /services/{serviceId}/usage:
+    get:
+      summary: Retrieve the API Info that use the given service
+      description: |
+        Retrieve the id, name, context and version of the APIs that used by the service
+      security:
+        - OAuth2Security:
+            - service_catalog:service_view
+      operationId: getServiceUsage
+      tags:
+        - Services
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+      responses:
+        200:
+          description: |
+            List of APIs that uses the service in the service catalog is returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/APIList'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+                "https://127.0.0.1:9443/api/am/service-catalog/v0/services/3784c2da-2f2a-4af9-8a43-ef207fb4d739/usage"'
+
+  #-------------------------------------------------------------------
+  # Import a service into the service catalog from a zipped archive
+  #-------------------------------------------------------------------
+  /services/import:
+    post:
+      summary: Import a service
+      description: |
+        Import  a service by providing an archived service
+      security:
+        - OAuth2Security:
+            - service_catalog:service_write
+      operationId: importService
+      tags:
+        - Services
+      parameters:
+        - name: overwrite
+          in: query
+          description: |
+            Whether to overwrite if there is any existing service with the same name and version.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      requestBody:
+        required: false
+        content:
+          multipart/form-data:
+            schema:
+              required:
+                - file
+              properties:
+                file:
+                  type: string
+                  description: |
+                    Zip archive consisting of exported Application Configuration.
+                  format: binary
+                verifier:
+                  type: string
```

Comment:
Shall we put a similar explanation to the verifier field too?

Reply:
Fixed in 0e2d05ca1eb804b7d1352c77d474e1c01accdbdd